app/views/components/link.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui.HyperLink
+
+@(link: HyperLink)(implicit messages: Messages)
+
+<a class="govuk-link" href="@link.dest" @{if (link.window) "target=_blank"}>@link.txt.value(messages.lang)</a>
```

Comment:
Should the target definition be target="_blank" rather than "target=_blank"

Reply:
I started with target="_blank"  and found it produced multiple quotes, the "target=_blank" syntax with the if statement produced the correct result